e2e/cypress/integration/channel/message_reply_no_pretext_spec.js

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import users from '../../fixtures/users.json';
+
+const sysadmin = users.sysadmin;
+
+describe('Message Reply to bot message', () => {
+    let newChannel;
+    let botsUserId;
+    before(() => {
+        // # Set ServiceSettings to expected values
+        const newSettings = {
+            ServiceSettings: {
+                EnableBotAccountCreation: true,
+            },
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        // # Login and go to /
+        cy.apiLogin('sysadmin');
+
+        // # Create and visit new channel
+        cy.createAndVisitNewChannel().then((channel) => {
+            newChannel = channel;
+        });
+    });
+
+    it('MM-16735 Reply to an older bot post that has no content and no attachment pretext', () => {
```

Comment:
```suggestion
    it('M16735 - Replying to an older bot post that has no post content and no attachment pretext', () => {
```

Reply:
For the [other test](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/integration/channel/message_reply_pretext_spec.js#L35), could please change test description to "M16734 - Replying to an older bot post that has some attachment pretext before the message attachment"?